src/inputs/input-transformers.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'underscore';
+
+let normalizeItemsOption = (input) => {
+    if (!input.options.items) {
```

Comment:
Yea, thought about that and chose to optimistically do it for all inputs (because the chances of an input having a `-items` option that wouldn't have this behavior seems low). I also wanted to avoid explicitly calling out input types here. I think the more correct way to do it would be as you described...we can have a set of transformers and then each individual input can pick and choose which transformers it wants to use. Do you think this is worth doing here/now or leaving it up to juttle to add support for this stuff?